src/ens.ts

```diff
@@ -1,105 +1,135 @@
-import {default as ensInterface} from './ens/contract/ens';
-import {default as registrarInterface} from './ens/contract/registrar';
-import {default as deedInterface} from './ens/contract/deed';
-import {default as resolverInterface} from './ens/contract/resolver';
-import {hash} from 'eth-ens-namehash';
+import { default as ensInterface } from './ens/contract/ens';
+import { default as registrarInterface } from './ens/contract/registrar';
+import { default as deedInterface } from './ens/contract/deed';
+import { default as resolverInterface } from './ens/contract/resolver';
+import { hash } from 'eth-ens-namehash';
 
 const Web3 = require('web3');
 
 const BLANK_ADDRESS = '0x0000000000000000000000000000000000000000';
 const DEFAULT_SOURCE = 'wss://mainnet.infura.io/ws';
+const DEFAULT_OPTIONS = {
+    isTest: false,
```

Comment:
Assuming the option appeared because `nock` doesn't support websockets.
I think we should just avoid using websockets in our test suite. `Web3` supports http(s) instead so we can  leave the `DEFAULT_SOURCE` as a websocket and set URL like this in ENS blockchain tests:

``` js
new Namicorn(blockchain: {ens: 'https://mainnet.infura.io'})
```